cmd/gabi/exec/cmd.go

```diff
@@ -63,7 +70,19 @@ func run(cmd *cobra.Command, argv []string) {
 
 	var query string
 	if len(argv) > 0 {
-		query = argv[0]
+		// If the given arguments is a path, attempt reading the file. Otherwise, assume a query string was given.
+		if _, fileExistsErr := os.Stat(argv[0]); fileExistsErr == nil {
+			body, readFileErr := ioutil.ReadFile(argv[0])
+			if readFileErr != nil {
+				logErrAndExit(readFileErr.Error())
+			}
+
+			log.Debugf(`Query read from '%s'`, argv[0])
+
+			query = string(body)
+		} else {
+			query = argv[0]
+		}
```

Comment:
The problem with this approach is that if the file does not exist, the path is taken as the query to be sent to Gabi, and of course the database complains about some filename not being valid syntax.